atm.js

```diff
@@ -1,21 +1,73 @@
 var checkingBalance = document.querySelector("div#checking_balance");
 checkingBalance = parseInt( checkingBalance.innerHTML.replace("$", "") );
 
-var checkingDeposit = function(amount){
+checkingDeposit = function(amount){
   checkingBalance = parseInt( checkingBalance.innerHTML.replace("$", "") );
+};
+// var checkingBalance = document.querySelector(".checking_balance").innerHTML;
+var extraAmount = 0;
 
-}
+checkingDeposit = function(){
+  inputAmount = parseInt($('.inputField').val());
+  checkingBalance = parseInt( document.querySelector(".checking_balance").innerHTML.replace("$", "") );
+  balanceAmount = inputAmount.checkingBalance;
```

Comment:
I'm not sure what `balanceAmount = inputAmount.checkingBalance `is suppose to be doing in this case... You have already declared` inputAmount` and `checkingBalance` above as variables and cannot use dot notation on variables. Also, just remember to watch out for global variables and use `var inputAmount`